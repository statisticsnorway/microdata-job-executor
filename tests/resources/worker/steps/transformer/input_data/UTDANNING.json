{
  "temporalityType": "STATUS",
  "sensitivityLevel": "PERSON_GENERAL",
  "populationDescription": [
    {
      "languageCode": "no",
      "value": "Personer med utdannelse i Norge"
    }
  ],
  "subjectFields": [
    [
      {
        "value": "BEFOLKNING",
        "languageCode": "no"
      }
    ],
    [
      {
        "value": "UTDANNING",
        "languageCode": "no"
      }
    ]
  ],
  "spatialCoverageDescription": [
    {
      "languageCode": "no",
      "value": "Norge"
    }
  ],
  "dataRevision": {
    "description": [
      {
        "languageCode": "no",
        "value": "Første publisering."
      }
    ],
    "temporalEndOfSeries": false,
    "temporalCoverageStart": "1990-01-01",
    "temporalCoverageLatest": "1994-01-01",
    "temporalStatusDates": [
      "1990-01-01",
      "1991-01-01",
      "1992-01-01",
      "1993-01-01",
      "1994-01-01"
    ]
  },
  "measureVariables": [
    {
      "name": [
        {
          "languageCode": "no",
          "value": "Utdanning"
        }
      ],
      "description": [
        {
          "languageCode": "no",
          "value": "Utdanningsnivå for person"
        }
      ],
      "subjectFields": [
        [
          {
            "languageCode": "no",
            "value": "Testdata"
          }
        ]
      ],
      "dataType": "STRING",
      "valueDomain": {
        "codeList": [
          {
            "code": "1",
            "categoryTitle": [
              {
                "languageCode": "no",
                "value": "Grunnskole"
              }
            ],
            "validFrom": "1926-01-01",
            "validUntil": null
          },
          {
            "code": "2",
            "categoryTitle": [
              {
                "languageCode": "no",
                "value": "Videregående skole"
              }
            ],
            "validFrom": "1926-01-01",
            "validUntil": null
          },
          {
            "code": "3",
            "categoryTitle": [
              {
                "languageCode": "no",
                "value": "Bachelorgrad"
              }
            ],
            "validFrom": "1926-01-01",
            "validUntil": null
          },
          {
            "code": "4",
            "categoryTitle": [
              {
                "languageCode": "no",
                "value": "Mastergrad"
              }
            ],
            "validFrom": "1926-01-01",
            "validUntil": null
          },
          {
            "code": "5",
            "categoryTitle": [
              {
                "languageCode": "no",
                "value": "Doktorgrad"
              }
            ],
            "validFrom": "1926-01-01",
            "validUntil": null
          }
        ]
      },
      "shortName": "UTDANNING"
    }
  ],
  "shortName": "UTDANNING"
}