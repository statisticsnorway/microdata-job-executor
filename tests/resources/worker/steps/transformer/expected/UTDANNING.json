{
    "name": "UTDANNING",
    "populationDescription": "Personer med utdannelse i Norge",
    "languageCode": "no",
    "temporality": "STATUS",
    "sensitivityLevel": "PERSON_GENERAL",
    "subjectFields": [
        "BEFOLKNING",
        "UTDANNING"
    ],
    "temporalCoverage": {
        "start": 7305,
        "stop": 8766
    },
    "identifierVariables": [
        {
            "variableRole": "Identifier",
            "name": "PERSONID_1",
            "label": "Personidentifikator",
            "notPseudonym": false,
            "dataType": "Long",
            "representedVariables": [
                {
                    "description": "Identifikator for person i microdata",
                    "validPeriod": {
                        "start": 7305,
                        "stop": 8766
                    },
                    "valueDomain": {
                        "description": "Pseudonymisert personnummer",
                        "unitOfMeasure": "N/A"
                    }
                }
            ],
            "format": "RandomUInt48",
            "keyType": {
                "name": "PERSON",
                "label": "Person",
                "description": "Person er et enkeltmenneske, individ."
            }
        }
    ],
    "measureVariable": {
        "variableRole": "Measure",
        "name": "UTDANNING",
        "label": "Utdanning",
        "notPseudonym": true,
        "dataType": "String",
        "representedVariables": [
            {
                "description": "Utdanningsniv\u00e5 for person",
                "validPeriod": {
                    "start": -16071
                },
                "valueDomain": {
                    "codeList": [
                        {
                            "category": "Grunnskole",
                            "code": "1"
                        },
                        {
                            "category": "Videreg\u00e5ende skole",
                            "code": "2"
                        },
                        {
                            "category": "Bachelorgrad",
                            "code": "3"
                        },
                        {
                            "category": "Mastergrad",
                            "code": "4"
                        },
                        {
                            "category": "Doktorgrad",
                            "code": "5"
                        }
                    ],
                    "missingValues": []
                }
            }
        ]
    },
    "attributeVariables": [
        {
            "variableRole": "Start",
            "name": "START",
            "label": "Startdato",
            "notPseudonym": true,
            "dataType": "Instant",
            "representedVariables": [
                {
                    "description": "Startdato/m\u00e5letidspunktet for hendelsen",
                    "validPeriod": {
                        "start": 7305,
                        "stop": 8766
                    },
                    "valueDomain": {
                        "description": "Dato oppgitt i dager siden 1970-01-01",
                        "unitOfMeasure": "N/A"
                    }
                }
            ]
        },
        {
            "variableRole": "Stop",
            "name": "STOP",
            "label": "Stoppdato",
            "notPseudonym": true,
            "dataType": "Instant",
            "representedVariables": [
                {
                    "description": "Stoppdato/m\u00e5letidspunktet for hendelsen",
                    "validPeriod": {
                        "start": 7305,
                        "stop": 8766
                    },
                    "valueDomain": {
                        "description": "Dato oppgitt i dager siden 1970-01-01",
                        "unitOfMeasure": "N/A"
                    }
                }
            ]
        }
    ],
    "temporalStatusDates": [
        7305,
        7670,
        8035,
        8401,
        8766
    ]
}